packages/data-transport-layer/src/services/l2-ingestion/service.ts

```diff
@@ -184,13 +184,17 @@ export class L2IngestionService extends BaseService<L2IngestionServiceOptions> {
           true,
         ],
         id: '1',
-      };
+      }
 
-      const resp = await axios.post(this.state.l2RpcProvider.connection.url, req, {responseType: 'stream'});
-      const respJson = await bfj.parse(resp.data, { 
-        yieldRate: 4096 // this yields abit more often than the default of 16384
-      });
-      const blocks = respJson.data;
+      const resp = await axios.post(
+        this.state.l2RpcProvider.connection.url,
+        req,
+        { responseType: 'stream' }
+      )
+      const respJson = await bfj.parse(resp.data, {
+        yieldRate: 4096, // this yields abit more often than the default of 16384
+      })
+      blocks = respJson.data
```

Comment:
Does removing the `const` from `const blocks = respJson.data;` warrant a changeset? Was this a bug?

Reply:
It does not since there's no functionality change.

Reply:
This was indeed a bug! Fixed here: https://github.com/ethereum-optimism/optimism/pull/1037